### DISCUSSION TITLE
Welcome to t490-macos Discussions!

### DISCUSSION THREAD

**Discussion by yusifsalam** [yusifsalam/t490-macos#33] [General]
## 👋 Welcome!
Here you can ask quick questions that don't necessarily fit within issues.

**Comment by kewave** [comment]
How to unlock CFG of T490???

**Reply by yusifsalam** [reply]
https://dortania.github.io/OpenCore-Post-Install/misc/msr-lock.html

**Comment by kewave** [comment]
i followed this method, it can not unlock because the motherboard of t490 is special (insyde)发自我的华为手机-------- 原始邮件 --------发件人： Yusif Salam-zade <notifications@github.com>日期： 2021年2月19日周五 18:38收件人： yusifsalam/t490-macos <t490-macos@noreply.github.com>抄送： kewave <kewave@qq.com>, Comment <comment@noreply.github.com>主    题： Re: [yusifsalam/t490-macos] Welcome to t490-macos Discussions! (#33)
https://dortania.github.io/OpenCore-Post-Install/misc/msr-lock.html

**Comment by rodo89usa17** [comment]
Hello, I am sorry the noob question. How can I make use of this repository to install macOS big sur in my T490?. Thank you in advance.

**Reply by yusifsalam** [reply]
README has the instructions https://github.com/yusifsalam/t490-macos#pre-install-creating-the-installation-usb-stick

You should first read the Dortania OC guide from here https://dortania.github.io/OpenCore-Install-Guide/

**Comment by Ngdaz** [comment]
It works perfect for me with t490 (version i5-8365U)
thank you so much
![180857420_666980494133732_4070527904076318072_n](https://user-images.githubusercontent.com/47729815/117232673-d38dd500-ae4b-11eb-9af7-f4daf6bee504.png)

**Reply by Ngdaz** [reply]
How can i use battery threshold ?

**Reply by yusifsalam** [reply]
i'm not sure what you mean

**Reply by Ngdaz** [reply]
I want to stop charge my laptop at 80% like in windows with Lenovo vintage, I can set up it

**Reply by Ngdaz** [reply]
I have read that I can do it with YogaSMC. But I don't know how to install it, is there any guild for me to follow or read

**Reply by yusifsalam** [reply]
You need to install the YogaSMCNC app separately https://github.com/zhen-zen/YogaSMC#yogasmcnc. Download it from the releases [here](https://github.com/zhen-zen/YogaSMC/releases/download/1.4.3/YogaSMC-App-Release.dmg), install it, and it should work without needing to change anything else in the config.

**Reply by Ngdaz** [reply]
Oh, it works, thank you

**Reply by Ngdaz** [reply]
Still have a problem with setting the threshold number :(((, the setting reset to 0 and 100 after turn off SMCpane, and it doesn't work with the command line: "ioio -s ThinkVPC setCMstart 75", did i config it wrong
![image](https://user-images.githubusercontent.com/47729815/117530953-2f9e5800-b00a-11eb-8a7b-b5ce42827c07.png)

**Reply by yusifsalam** [reply]
From what I see in the screenshot, you don't have ioio installed or it's not in PATH so zsh can't find it.

**Reply by Ngdaz** [reply]
After install ioio, then using the command line to set the battery threshold(because the yogaSMCPane didn't work), now my laptop can stop the charge at 80 and start at 75, thank you.
![image](https://user-images.githubusercontent.com/47729815/117843576-22eb5f80-b2a9-11eb-859c-98f925999d78.png)

**Comment by 1074566721** [comment]
你好，我是t590的用户，我用你提供的oc引导安装在t590上wifi信号有时会消失，非常不稳定，这是t490也存在的问题吗？

**Reply by jaredev** [reply]
Just wanted to leave a quick comment saying thanks! I was able to use the m.2 drive from my desktop, replace it in the ThinkPad T490 and have it booting just by replacing the EFI files. So far, everything works fine using macOS Catalina 10.15.7 and the catalina release of the files on the repo here. Wake from sleep works, battery charging, trackpad, keyboard, wifi, bluetooth, and so on. Great work! Thanks again.

**Reply by yusifsalam** [reply]
Thanks! If you're not going to update to Big Sur, you should at least update the kexts, because there was a pretty decent wifi speed improvement  at some point after the Catalina release.
